include/OscapScannerRemoteSsh.h

```diff
@@ -36,12 +36,18 @@ class OscapScannerRemoteSsh : public OscapScannerBase
         OscapScannerRemoteSsh();
         virtual ~OscapScannerRemoteSsh();
 
+        void setUserIsSudoer(bool userIsSudoer);
         virtual void setTarget(const QString& target);
         virtual void setSession(ScanningSession* session);
 
         virtual QStringList getCommandLineArgs() const;
         virtual void evaluate();
 
+    protected:
+
+       virtual void selectError(MessageType& kind, const QString& message);
```

Comment:
```
In file included from /home/jenkins/workspace/scap-workbench-pull-requests/src/MainWindow.cpp:24:
/home/jenkins/workspace/scap-workbench-pull-requests/include/OscapScannerRemoteSsh.h:48:33: error: ‘MessageType’ has not been declared
        virtual void selectError(MessageType& kind, const QString& message);
                                 ^~~~~~~~~~~
```

Reply:
This fails because of the dependency on #271